storage/include/storage/db_interface.h

```diff
@@ -73,6 +73,7 @@ class IDBClient {
   virtual Status rangeDel(const Sliver& _beginKey, const Sliver& _endKey) = 0;
   virtual bool isNew() = 0;
   virtual Status createCheckpoint(const uint64_t&) { return Status::IllegalOperation("Not supported"); };
+  virtual std::string getPath() const { return ""; };
```

Comment:
return "NotSupported";
for better error handling if required.

Reply:
Since, this returns path in the file system and we always check if path exist. So, in the negative case, path with empty string will always fail the validity check. So, i have kept it this way